### PR TITLE
Create a dynamic_fields_group form builder method

### PR DESCRIPTION
**Pull request by fedegl**
Add a form builder method for adding associated records in a form.

### CODE REVIEW

File: lib/frontend_helpers/form_builder_helpers/dynamic_fields.rb [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      #   <%= f.dynamic_fields_group :items do %>
+      #     <label>Items</label>
+      #     <%= f.link_to_add_fields "Add Item", :items %>
+      #   <% end %>
+      #
+      # Requirements:
+      # - Partial for the associated fields with the name:
+      #     "_#{singluar_association_name}_fields.html.erb"
+      # - The template needs to have a top level element with the class:
+      #     "#{singluar_association_name}-fields"
```
Comment (alexisKinto) [inline lib/frontend_helpers/form_builder_helpers/dynamic_fields.rb:26]: #     "_#{singular_association_name}_fields.html.erb"
      # - The template needs to have a top level element with the class:
      #     "#{singular_association_name}-fields"

**Review by rafaelcdelgado — APPROVED** [review]
🚀

File: lib/frontend_helpers/form_builder_helpers/dynamic_fields.rb [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+          safe_join([
+                      label_tag(method, options),
+                      add_link_tag(method, options)
+                    ])
+        end
+      end
+
+      def label_tag(method, options)
+        translated_label = I18n.t(
+          "activerecord.attributes.#{object.model_name.i18n_key}.#{method}"
```
Comment (alexisKinto) [inline lib/frontend_helpers/form_builder_helpers/dynamic_fields.rb:116]: Si no encuentra traducción nos marcaria error?

**Review by alexisKinto — CHANGES_REQUESTED** [review]

**Review by alexisKinto — APPROVED** [review]

File: lib/frontend_helpers/form_builder_helpers/dynamic_fields.rb (hunk 2 above)
Comment (fedegl) [reply lib/frontend_helpers/form_builder_helpers/dynamic_fields.rb:116]: va a mostrar un mensaje que no existe la traducción

**Pull request merged**